templates/kubernetes/resources.html

```diff
@@ -0,0 +1,490 @@
+        <p>This webinar demonstrates how Canonical's modular technology stack addresses these challenges, using well-known cloud products such as MicroK8s and LXD.</p>
+      </div>
+      <div class="col-4">
+        <div class="u-hide--small u-align--center">
+          <a href="https://ubuntu.com/engage/container-orchestration-edge">
+            <img src="https://www.brighttalk.com/communication/470152/preview_1613133266.png" alt="Micro clouds: Secure container orchestration at the edge" width="560" height="302">
+           </a>
+        </div>
+      </div>
+    </div>
+  </article>
+  <hr class="is-muted is-fixed-width" />
+  <article class="p-strip is-shallow">
+    <div class="row">
+      <div class="col-4">
+        <h3 class="p-heading--4"><a href="https://ubuntu.com/engage/edge-kubernetes-raspberry-pi-webinar">K8s at the edge: easy as "Pi"</a></h3>
+      </div>
+      <div class="col-4">
+        <p>Discover how to build your own home cloud in this step-by-step video tutorial which combines the power of technologies such as LXD, MicroK8s, Charmed Operators, and Raspberry Pis.</p>
+      </div>
+      <div class="col-4">
+        <div class="u-hide--small u-align--center">
+          <a href="https://ubuntu.com/engage/edge-kubernetes-raspberry-pi-webinar">
+            <img src="https://www.brighttalk.com/communication/488796/preview_1621331611.png" alt="K8s at the edge" width="560" height="302">
+           </a>
+        </div>
+      </div>
+    </div>
+  </article>
+</section>
+<section class="p-strip--light">
+  <div class="u-fixed-width">
+    <h2>Case studies/Whitepapers/Industry reports</h2>
+  </div>
+  <article class="p-strip is-shallow">
+    <div class="row">
+      <div class="col-4">
+        <h3 class="p-heading--4"><a href="/engage/atresmedia-charmed-kubernetes-case-study">Atresmedia accesses high availability and unprecedented productivity with Charmed Kubernetes</a></h3>
+      </div>
+      <div class="col-4">
+        <p>Atresmedia, the Spanish OTT media service provider responsible for hits shows such as La Casa de Papel (Money Heist), has been the most loyal customer of Canonical Kubernetes throughout the years.</p>
+        <p>In this case study, you will learn how the company reached its technical and growth acceleration goals by hosting its Atresplayer platform leveraging Charmed Kubernetes.</p>
+      </div>
+      <div class="col-4">
+        <div class="u-hide--small u-align--center">
+          <a href="/engage/atresmedia-charmed-kubernetes-case-study">
+            <img src="https://ubuntu.com/wp-content/uploads/89d1/Atresmedia.png" alt="Atresmedia accesses high availability and unprecedented productivity with Charmed Kubernetes" width="560" height="302">
```

Comment:
Optional comment. Shouldn't we have this in assets?

Reply:
This is the one Alex provided in the copy doc. You mean we have the same image in the asset? I didn't know that!